apis/python/src/tiledbsoma/_tdb_handles.py

```diff
@@ -194,20 +243,37 @@ def _opener(
     def schema(self) -> tiledb.ArraySchema:
         return self._handle.schema
 
-    def non_empty_domain(self) -> Tuple[Tuple[int, int], ...]:
-        """
-        Retrieves the non-empty domain for each dimension, namely the smallest
-        and largest indices in each dimension for which the array/dataframe has
-        data occupied.  This is nominally the same as the domain used at
-        creation time, but if for example only a portion of the available domain
-        has actually had data written, this function will return a tighter
-        range.
-        """
-        return self._handle.nonempty_domain()  # type: ignore
+    def non_empty_domain(self) -> Optional[Tuple[Tuple[Any, Any], ...]]:
```

Comment:
I think you may have already responded to this and I missed it, but I had some previous discussion on the merits of `object` vs. `Any` either in this change or one of the similar changes which I cannot find at the moment. Just wanted to bring it up again in case not; if you already have that’s fine.